crates/bevy_ecs/macros/src/fetch.rs

```diff
@@ -255,7 +261,10 @@ pub fn derive_world_query_impl(ast: DeriveInput) -> TokenStream {
     let fetch_impl = impl_fetch(false, fetch_struct_name.clone(), item_struct_name.clone());
 
     let state_impl = quote! {
-        #[doc(hidden)]
+        #[doc = "Item for ["]
```

Comment:
"Automatically generated internal [`WorldQuery`] state type for [`Foo`], used for caching."